<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>294</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>Add New Item</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resource.qrc" >
    <normaloff>:/newPrefix/logo.jpg</normaloff>:/newPrefix/logo.jpg</iconset>
  </property>
  <property name="toolTip" >
   <string>Add New Item</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>346</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text" >
    <string>Give the co-ordinates of the vertices of the polygon with centre at (0,0)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="X_Vertex" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>60</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="Y_Vertex" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>60</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddVertex" >
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>60</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Add New Vertex</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>80</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>250</y>
     <width>208</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text" >
    <string>Note:The item rotates with respect to (0,0)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>80</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>Vertex Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>80</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddItem" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="VertexNumber" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>60</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc" />
 </resources>
 <connections/>
</ui>
